kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListView.java

```diff
@@ -410,12 +402,29 @@ public HTMLDivElement getListItems() {
     }
 
     @Override
-    public void showImportDataObjectButton() {
-        show(importDataObjectButton);
+    public void renderImportJavaClasses() {
+        if (!importJavaClassesContainer.hasChildNodes()) {
+            renderImportJavaClasses(".kie-import-java-classes",
+                                              false,
+                                              null);
+        }
     }
 
-    @Override
-    public void hideImportDataObjectButton() {
-        hide(importDataObjectButton);
+    /**
+     * Indirection for Test purposes only
+     */
+    protected void renderImportJavaClasses(final String selector,
+                                           final boolean buttonDisabledStatus,
+                                           final String buttonTooltipMessage) {
+        DMNLoader.renderImportJavaClasses(selector,
+                                          buttonDisabledStatus,
+                                          buttonTooltipMessage);
+    }
+
+    /**
+     * Indirection for Test purposes only
+     */
+    protected void registerBroadcastForImportJavaClasses() {
+        ImportJavaClassesService.registerBroadcastForImportJavaClasses(this);
     }
```

Comment:
Please, help me to understand better why you need these two methods.

Reply:
@vpellegrino That is a common solution already used in this repo to verify a static method is called. In this way, you can verify a static method is called without mocking it. Of course, It is possible to mock the static method using PowerMockito or the latest Mockito API, but that requires correctly setting up all required preconditions (eg. Annotations) but I preferred to use this indirection solution to simplify the testing process.  Aware is not ideal to modify the regular code to handle tests - so I can try to reconsider using the api. I'll let u know.

Reply:
@yesamer fine for me to keep things as are.
No need to invest more time on this from my point of view.
Thanks for the clarification